CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+
+### dbt Labs contributors
+
+If you are a member of the `dbt-labs` GitHub organization, you will have push access to the `actions` repo. Rather than forking `actions` to make your changes, just clone the repository, check out a new branch, and push directly to that branch.
+
+
+### Tools
+
+These are the tools used in `actions` development and testing:
+
+- [`github workflows`](https://docs.github.com/en/actions/using-workflows) for shared workflows
+- [`Understanding Github Actions`](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions)
+
+A deep understanding of these tools in not required to effectively contribute to `actions`, but we recommend checking out the attached documentation if you're interested in learning more about each one.
+
+## Running `actions` in development
+
+### Installation
+
+None
+
+### Running `actions`
+
+- Each workflow can be run indepndently, depending on its trigger
+- See list of trigger in the [GitHub docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)
+- It can be helpful to create a dummy repository to trigger actions and workflow
+
+
+### Initial setup
+
+None needed.
+
+### Test commands
+
+No tests included.
+
+
+## Debugging
+
+You can enable debug logging for GHA by setting secret values for your repository. See [docs](https://docs.github.com/en/github-ae@latest/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) for more info.
+
+- Set `ACTIONS_RUNNER_DEBUG` to `true` to enable runner diagnostic logging.
+- Set `ACTIONS_STEP_DEBUG` to `true` to enable run step debug logging.
+
+### Assorted development tips
+* TBD
+
+
+## Submitting a Pull Request
+
+Code can be merged into the current development branch `main` by opening a pull request. A `actions` maintainer will review your PR. They may suggest code revision for style or clarity, or request that you add unit or integration test(s). These are good things! We believe that, with a little bit of help, anyone can contribute high-quality code.
+
+Automated tests run via GitHub Actions. If you're a first-time contributor, all tests (including code checks and unit tests) will require a maintainer to approve. Changes in the `actions` repository trigger integration tests against Postgres. dbt Labs also provides CI environments in which to test changes to other adapters, triggered by PRs in those adapters' repositories, as well as periodic maintenance checks of each adapter in concert with the latest `actions` code changes.
+
+Once all tests are passing and your PR has been approved, a `actions` maintainer will merge your changes into the active development branch. And that's it! Happy developing :tada:
+
+Sometimes, the content license agreement auto-check bot doesn't find a user's entry in its roster. If you need to force a rerun, add `@cla-bot check` in a comment on the pull request.
```

Comment:
Is it worth adding a troubleshooting header? are there other tips for troubleshooting?

Reply:
This is a good idea to make these things more visible.  I can't count the number of time we're talking about how to do this cla-bot check.